finesse/config.py

```diff
@@ -8,3 +8,6 @@
 
 ANGLE_PRESETS = ("zenith", "nadir", "hot_bb", "cold_bb", "home")
 """Names of preset angles that the mirror can rotate to."""
+
+BAUDRATES = (4800, 9600, 19200, 38400, 57600, 115200)
```

Comment:
I guess I have a general question about how to best set and use the baud rates. 

I know that the DP9800 uses a baud rate of 38400 - presumably each of the other components have their own preferred or valid baud rate? I understand that these are just used to populate the list boxes, but do we not also want to check that when a baud rate is selected from the list, it corresponds to the correct baud rate for that instrument?

Reply:
Kind of. The thing is that the old GUI had this as an option and we're trying to replicate that for now. In theory you could change the baudrate for the device if you're reworking the hardware, in which case you would also want to change the value in the program, but this is hardly going to be a common occurrence! Having it as an option in the main window just increases the chance that someone will change it by accident then wonder why it's stopped working.

I think going forward we should hide this feature in a separate settings panel, with the settings defaulting to whatever the baudrate is for each device.